mappings/net/minecraft/world/HeightLimitView.mapping

```diff
@@ -1,17 +1,82 @@
 CLASS net/minecraft/class_5539 net/minecraft/world/HeightLimitView
-	METHOD method_31597 getTopSectionLimit ()I
-	METHOD method_31600 getTopHeightLimit ()I
+	COMMENT A view with a high limit specification.
+	METHOD method_31597 getTopSectionCoord ()I
+		COMMENT Returns the top section coordinate, exclusive, of this view.
+		COMMENT
+		COMMENT @implNote This implementation passes the {@linkplain #getTopY() top Y}
+		COMMENT through {@link net.minecraft.util.math.ChunkSectionPos#getSectionCoord(int)}.
+		COMMENT
+		COMMENT @return the top section coordinate
+		COMMENT @see #getBottomSectionCoord()
+		COMMENT @see #getTopY()
+	METHOD method_31600 getTopY ()I
+		COMMENT Returns the top Y level, or height, exclusive, of this view.
+		COMMENT
+		COMMENT @implNote This implementation sums up the bottom Y and the delta Y.
+		COMMENT @see #getBottomY()
+		COMMENT @see #getDeltaY()
 	METHOD method_31601 isOutOfHeightLimit (I)Z
+		COMMENT Checks if {@code y} is out of the height limit of this view.
+		COMMENT
+		COMMENT <P>{@code y} is out of bounds if it's lower than the {@linkplain #getBottomY
+		COMMENT bottom} or higher than or equal to the {@linkplain #getTopY() top}.
+		COMMENT
+		COMMENT @return {@code true} if {@code y} is out of bounds, {@code false} otherwise.
 		ARG 1 y
+			COMMENT the Y level to check
 	METHOD method_31602 getSectionIndex (I)I
+		COMMENT Returns a zero-based section index to which the {@code y} level belongs.
+		COMMENT
+		COMMENT @return a zero-based index
 		ARG 1 y
-	METHOD method_31603 getSectionIndexFromSection (I)I
-		ARG 1 section
-	METHOD method_31604 getSection (I)I
-		ARG 1 sectionIndex
-	METHOD method_31605 getSectionCount ()I
+	METHOD method_31603 sectionCoordToIndex (I)I
+		COMMENT Converts a section coordinate to a zero-based section index.
+		COMMENT
+		COMMENT @return a zero-based index
+		COMMENT @see #sectionIndexToCoord(int) the inverse operation <code>sectionIndexToCoord</code>
+		ARG 1 coord
+			COMMENT the section coordinate
+	METHOD method_31604 sectionIndexToCoord (I)I
+		COMMENT Converts a zero-based section index to a section coordinate.
+		COMMENT
+		COMMENT @return a section coordinate
+		COMMENT @see #sectionCoordToIndex(int) the inverse operation <code>sectionCoordToIndex</code>
+		ARG 1 index
+			COMMENT the zero-based section index
+	METHOD method_31605 getDeltaY ()I
```

Comment:
Maybe `getVerticalSize` or `getHeight`?

IMO `world.getDeltaY()` is unclear what `delta` means.

Reply:
getHeight sounds good. I need to document that mojang calls this `logical_height` in the dimension codec

Reply:
Turns out logical height is an unrelated concept only used by teleporting/portal logic. I renamed an associated method in world though (called `getHeightLimit` but calls `dimension.getLogicalHeight`)